draft-packages/table/KaizenDraft/Table/styles.scss

```diff
@@ -29,10 +29,6 @@ $row-height: 60px;
 .header {
   border: solid 1px $ca-border-color;
 
-  &.ash {
```

Comment:
For the curious. The new table headers should not have a background colour. I've also removed any colours hover states.